product_order_noname/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Vauxoo, S.A. de C.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Product Order No Name',
```

Comment:
I'd use "sort" instead of "order", because order could be confused with documents (e.g. sale/purchase orders), not with the actual sorting criteria. This also for the module name.

What do you think @moylop260 ?

Reply:
Even if you are right `order` is most used than sort in odoo.

I'm talking about `odoo.models.Models._order` attribute

So, I think it is a good name too in this context